compiler/rustc_span/src/symbol.rs

```diff
@@ -770,6 +770,7 @@ symbols! {
         impl_lint_pass,
         impl_macros,
         impl_trait_in_bindings,
+        import,
```

Comment:
Just a thought, but I wonder if we should not intern this symbol since it's only used in error recovery and the cost of interning at runtime when we're going to fail anyway basically doesn't matter. There might be some small cost to having the symbol always interned, but I'm not 100% sure of that. Just thought I'd throw that idea out there 😄

Reply:
Hm... my understanding is that interning a symbol is very cheap... I'll look into it.